src/storybook/decorators.tsx

```diff
@@ -8,6 +8,31 @@ import { ApolloProvider } from '@apollo/client'
 import { ApolloClient, InMemoryCache } from '@apollo/client'
 import { useForm, FormProvider, UseFormOptions } from 'react-hook-form'
 
+import { colors } from 'styles/theme'
+
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faMoon, faSun } from '@fortawesome/free-regular-svg-icons'
+
+import { ThemeProvider } from 'styled-components'
+
+export const DarkModeThemeToggler = (DecoratedStory: () => JSX.Element): JSX.Element => {
+  const [darkMode, setDarkMode] = React.useState(false)
+  const theme = colors(darkMode)
+
+  const handleDarkMode = (): void => setDarkMode((darkMode) => !darkMode)
+
+  return (
+    <>
```

Comment:
I think the background of the whole story should be the background of the app

Reply:
good idea. i'll make these changes in whichever PR is preferred #1624 or #1626